install/recorders.csv

```diff
@@ -1011,7 +1011,7 @@ Kinemetrics,FBA-ES-T-BASALT,BASALT,2024,PTOS,20,7,,0,0,2013-12-16T14:00:00Z,2013
 Kinemetrics,FBA-ES-T-BASALT,BASALT,2024,MLZ,21,60,,0,0,2014-03-25T00:05:00Z,2019-10-31T21:33:00Z
 Kinemetrics,FBA-ES-T-BASALT,BASALT,2024,MLZ,21,0,,0,0,2019-10-31T21:34:00Z,2024-02-15T00:09:00Z
 Kinemetrics,FBA-ES-T-BASALT,BASALT,2025,PTOS,20,7,,0,0,2013-11-18T22:47:00Z,2013-12-16T13:47:29Z
-Kinemetrics,FBA-ES-T-BASALT,BASALT,2025,RRKS,20,94,,0,0,2014-06-06T00:25:25Z,9999-01-01T00:00:00Z
+Kinemetrics,FBA-ES-T-BASALT,BASALT,2025,RRKS,20,94,,0,0,2014-06-06T00:25:25Z,2024-09-25T21:31:00Z
```

Comment:
Seems like the azimuth needs an update to `274`.

https://github.com/GeoNet/tickets/issues/16492#issuecomment-2381634041

Reply:
updated